master/buildbot/process/properties.py

```diff
@@ -83,6 +84,14 @@ def master(self, value):
         self._master = value
 
     @property
+    def buildProps(self):
```

Comment:
I dont think that we should do this.

If a renderer needs to use the steps, then it will need to call the data api to fetch the steps.
using 
```
props.master.data.get(('builders', props.getProperty('buildername'), 'builds', props.getProperty('buildernumber'), 'steps')
```

Reply:
I had to change it to 
`yield props.master.data.get(('builders', props.getProperty('buildername'), 'builds', props.getProperty('buildnumber'), 'steps'))`

but that works, will update the code. I'm still a little lost on why the step logs have an `h` or `e` before them. Is there any guide to this, or an easy way to strip that ?